bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/config/CameraConfiguration.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link CameraConfiguration} is responsible for holding configuration information for
+ * Camera devices
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@NonNullByDefault
+public class CameraConfiguration extends NAThingConfiguration {
+    public static final String ID = "id";
```

Comment:
Is this constant required ?
It is surprising to have it here while it concerns a field not defined here.